### DISCUSSION TITLE
Issues after starting up

### DISCUSSION THREAD

**Discussion by ghost** [intelowlproject/IntelOwl#1972] [Q&A]
I have IntelOwl mostly set up, but some issues arise after the application starts. Firstly, I get a lot of warnings like these:

![image](https://github.com/intelowlproject/IntelOwl/assets/110360903/7eba8eaa-1ec4-41f4-9b51-5e77a2d6eed2)

Also, when I try to login, I am only able to login once. If I click dashboard, it immediately logs me out and won't let me log back in. Before, I've been able to make it to the dashboard screen, but all the sections are filled with Server Error 500 or 502 codes. 

Please let me know if you know of any way to resolve these issues. Thanks!

**Comment by mlodic** [comment]
you can ignore the warnings, they come from some libraries that we didn't manage to silence.

Also, as stated in the doc [here](https://intelowl.readthedocs.io/en/latest/Installation.html#tl-dr), 500 status codes are normal at the first start. Just wait and retry few minutes later.
